### PR TITLE
Add license inquiry function using external API (Github, Clearly Defined)

### PR DESCRIPTION
**Pull request by riyenas0925**
## Description
* license inquiry function using github API (not oss version and download location)
* **github token is required for testing**
image

### result
image

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by riyenas0925** [comment]
[Github Test.xlsx](https://github.com/fosslight/fosslight/files/7561149/Github.Test.xlsx)

**Comment by riyenas0925** [comment]
# Test OSS List (github token is required for testing))
* [Exist DB.xlsx](https://github.com/fosslight/fosslight/files/7561391/Exist.DB.xlsx)
* [Clearly Defined Test.xlsx](https://github.com/fosslight/fosslight/files/7561392/Clearly.Defined.Test.xlsx)
* [Github Test.xlsx](https://github.com/fosslight/fosslight/files/7561393/Github.Test.xlsx)

**Comment by soimkim** [comment]
[SelfCheck-Report-1-test_20211125.xlsx](https://github.com/fosslight/fosslight/files/7600181/SelfCheck-Report-1-test_20211125.xlsx)
스크린샷 2021-11-25 오후 1 29 10
When I click Check License with above data, an error pops up and it loads infinitely.

image

File: src/main/java/oss/fosslight/common/DependencyType.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package oss.fosslight.common;
```
Comment (soimkim) [inline src/main/java/oss/fosslight/common/DependencyType.java:1]: Please add copyright, license text

**Review by soimkim — CHANGES_REQUESTED** [review]

File: src/main/java/oss/fosslight/common/ExternalServiceType.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
+package oss.fosslight.common;
```
Comment (soimkim) [inline src/main/java/oss/fosslight/common/ExternalServiceType.java:1]: Please add copyright, license text

**Review by soimkim — APPROVED** [review]

**Pull request merged**